src/cgame/rocket/rocketFormControlSelect.h

```diff
@@ -119,22 +117,24 @@ class CvarElementFormControlSelect : public Rml::ElementFormControlSelect, publi
 	void UpdateValue()
 	{
 		Rml::String cvarValue = Cvar::GetValue( cvar.c_str() );
+
 		for ( int i = 0; i < GetNumOptions(); ++i )
 		{
 			Rml::Element* e = GetOption(i);
 			Rml::String value = e->GetAttribute<Rml::String>( "value", Rml::String() );
 			if ( value == cvarValue )
 			{
-				// If the selection is already correct, do not force set the cvar.
-				ignore_value_change = GetSelection() != i;
+				ignore_value_change = true;
 				SetSelection( i );
+				ignore_value_change = false;
 				return;
 			}
 		}
 		// No matches...Let's add an option for our current value.
 		Add( Str::Format( "Custom Value (%s)", cvarValue ), cvarValue, 0, false );
-		SetSelection( 0 );
 		ignore_value_change = true;
+		SetSelection( 0 );
+		ignore_value_change = false;
```

Comment:
Sounds like this should be a helper function if we always have to do this

Reply:
Eh, we only do it twice, I like the rule of three :)